template/.eslintrc.js

```diff
@@ -34,12 +34,12 @@ module.exports = {
   ],
 
   globals: {
-    'ga': true, // Google Analytics
-    'cordova': true,
-    '__statics': true,
-    'process': true,
-    'Capacitor': true,
-    'chrome': true
+    'ga': writable, // Google Analytics
+    'cordova': writable,
+    '__statics': writable,
+    'process': writable,
+    'Capacitor': writable,
+    'chrome': writable
```

Comment:
I'm pretty sure that `writable` should be quoted as it's a string.

```suggestion
    'ga': 'writable', // Google Analytics
    'cordova': 'writable',
    '__statics': 'writable',
    'process': 'writable',
    'Capacitor': 'writable',
    'chrome': 'writable'
```